src/metaCommand.cxx

```diff
@@ -106,7 +107,7 @@ MetaCommand::SetOption(Option option)
 
 bool
 MetaCommand::SetOption(std::string        name,
-                       std::string        shortTag,
+                       const std::string&        shortTag,
```

Comment:
auto-formatting should be applied after this change

Reply:
```
git filter-branch -f --tree-filter  'git-clang-format --binary /home/johnsonhj/Dashboard/src/ITK-bld/clang-format-Linux  ' --  origin/master..                                                    1 ↵
Rewrite 2d6e371442da52432223cbf60ef2bc4172ef11a1 (1/37) (0 seconds passed, remaining 0 predicted)    clang-format did not modify any files
Rewrite a1c8086ba3569aff68f7e6144ddc629adacf823e (2/37) (2 seconds passed, remaining 35 predicted)    clang-format did not modify any files
Rewrite dc9c57d45eea3bdb4f6d983aac95b8554c8e3efe (2/37) (2 seconds passed, remaining 35 predicted)    clang-format did not modify any files
Rewrite 5c11b8db1b5d0b70c0a868c8f5bb3adc263af940 (4/37) (6 seconds passed, remaining 49 predicted)    clang-format did not modify any files
Rewrite 1060a9628ef486acfa271d441d8075ae10dcb123 (5/37) (8 seconds passed, remaining 51 predicted)    clang-format did not modify any files
Rewrite 7c9c0e79cd23314cf26d21f9bde09c6cdae78812 (6/37) (9 seconds passed, remaining 46 predicted)    clang-format did not modify any files
Rewrite 73eeee7f1bea9816dd780516b161a2ca2a8d6f96 (7/37) (11 seconds passed, remaining 47 predicted)    clang-format did not modify any files
Rewrite 67cefa4ed4b49d54eb6e767d0f28cbd913863ce2 (8/37) (12 seconds passed, remaining 43 predicted)    clang-format did not modify any files
Rewrite 5bb3b4da72c7b0ef5596fa41d6cc08c002dec019 (9/37) (14 seconds passed, remaining 43 predicted)    clang-format did not modify any files
Rewrite 336c99d382bab223d1f3a6832b600a56acb92a65 (10/37) (15 seconds passed, remaining 40 predicted)    clang-format did not modify any files
Rewrite 062d6d2a58aa29e725022c681add868cd26b92ad (11/37) (17 seconds passed, remaining 40 predicted)    clang-format did not modify any files
Rewrite 54b977d9c90aa9165b7656d2707a15e7eb6ac9bf (12/37) (18 seconds passed, remaining 37 predicted)    clang-format did not modify any files
Rewrite b9b12933c48e7c31f69a2a6b8ef5ef489c804729 (13/37) (20 seconds passed, remaining 36 predicted)    clang-format did not modify any files
Rewrite 88a1523a3d6712504f08604390ee699d966c2d32 (14/37) (21 seconds passed, remaining 34 predicted)    clang-format did not modify any files
Rewrite 6967e6d160c59d02e9ee91a7978256268649cbdd (15/37) (23 seconds passed, remaining 33 predicted)    clang-format did not modify any files
Rewrite 632a827b5bac695a366dfc3b61f1b7008cf996b1 (16/37) (24 seconds passed, remaining 31 predicted)    clang-format did not modify any files
Rewrite 48650b52d864cdc69d9c0bf2acad6d02b8e16d55 (17/37) (25 seconds passed, remaining 29 predicted)    clang-format did not modify any files
Rewrite 58289683c0b130eaa321fe30bcc0407bd04b1461 (18/37) (27 seconds passed, remaining 28 predicted)    clang-format did not modify any files
Rewrite a847defff16d6134b65942b54a2482c8525f822a (19/37) (28 seconds passed, remaining 26 predicted)    clang-format did not modify any files
Rewrite a1ed37bb1e509497276133a1cf047a7f18bb0a19 (20/37) (30 seconds passed, remaining 25 predicted)    clang-format did not modify any files
Rewrite 36b54127445c21ba4417f2e024d3d9c5d1759fe2 (21/37) (31 seconds passed, remaining 23 predicted)    clang-format did not modify any files
Rewrite cf9488591287f0bc92a1042df7d6cf321e7a45a8 (22/37) (32 seconds passed, remaining 21 predicted)    clang-format did not modify any files
Rewrite fd03c02596c3ce201288627f80fb085db152c7ae (23/37) (34 seconds passed, remaining 20 predicted)    clang-format did not modify any files
Rewrite 900f57fd3f1d4811038ccd6ed8016c7f1da24e3a (24/37) (35 seconds passed, remaining 18 predicted)    clang-format did not modify any files
Rewrite f52d3838472d2f5394a4ffe4cd307015cfe099b8 (25/37) (36 seconds passed, remaining 17 predicted)    clang-format did not modify any files
Rewrite 81dc1a5c1f782966e0e580e1a4146d680cfd7843 (26/37) (37 seconds passed, remaining 15 predicted)    clang-format did not modify any files
Rewrite b1d789b282f6c0aeeabfea4df56f08cf9d08788e (27/37) (39 seconds passed, remaining 14 predicted)    clang-format did not modify any files
Rewrite 3cef3fccf92b09bc57f5253a1baa5444dfba8f6a (28/37) (40 seconds passed, remaining 12 predicted)    clang-format did not modify any files
Rewrite 3145b4c655af51f22d20117f067da6b9792ee6ff (29/37) (41 seconds passed, remaining 11 predicted)    clang-format did not modify any files
Rewrite 674c888c9661da76f15d416ad287d016ce118037 (30/37) (42 seconds passed, remaining 9 predicted)    clang-format did not modify any files
Rewrite 829514b80846a0cdc533b2ea6ac3d8bd696b4c52 (31/37) (43 seconds passed, remaining 8 predicted)    clang-format did not modify any files
Rewrite 1abeee73a5099dba109cc5169fe797aaf5987b6a (32/37) (44 seconds passed, remaining 6 predicted)    clang-format did not modify any files
Rewrite 823f2603bbb539846a14df25e92f93c096cb797b (33/37) (45 seconds passed, remaining 5 predicted)    clang-format did not modify any files
Rewrite 571d39e1733888373f82af115b7ee29b51f7b9cc (34/37) (46 seconds passed, remaining 4 predicted)    clang-format did not modify any files
Rewrite 07c6317743ff92c7deaf82e02aff65882e22fac2 (35/37) (47 seconds passed, remaining 2 predicted)    clang-format did not modify any files
Rewrite 32068dd7b4ebca6de87e76240d61c24a9dd56abd (36/37) (48 seconds passed, remaining 1 predicted)    clang-format did not modify any files
Rewrite cbdba79fcf68965f7c57e9d7d6ad30bf7870ae75 (37/37) (49 seconds passed, remaining 0 predicted)    no modified files to format

WARNING: Ref 'refs/heads/clang-tidy-code-cleanups' is unchanged


```